### ISSUE TITLE
Invalid input directory: `src`

### ISSUE THREAD

**Issue by shubie** [elixir-griffin/griffin#15]
Hi,

Followed the instructions on the quick start guide: https://hexdocs.pm/griffin_ssg/quick_start.html

I ran the command `mix grf.build` and I got Invalid input directory: `src`

What could be the cause

**Comment by gonglexin** [comment]
You need to make a new directory (src) in the blog folder and move the `README.md` file to the newly created directory.

Based on my tests, HTML files doesn't seem to work, but the EEx templates and markdown works fine.

**Comment by maikelthedev** [comment]
> You need to make a new directory (src) in the blog folder and move the `README.md` file to the newly created directory.
> 
> Based on my tests, HTML files doesn't seem to work, but the EEx templates and markdown works fine.

That is correct, as seen below, but that should be written in the docs otherwise [this page](https://hexdocs.pm/griffin_ssg/quick_start.html) is confusing: 

![Image](https://github.com/user-attachments/assets/e4faedd9-c044-4f55-9801-8b695e8fef98)
